holdings_maintenance/pds4/pds4checksums.py

```
@@ -0,0 +1,895 @@
+        raise
+
+    finally:
+        (fatal, errors, warnings, tests) = logger.close()
+        if fatal or errors:
+            proceed = False
+
+    # If everything went well, execute pdsinfoshelf too
+    if proceed and args.infoshelf:
+        new_list = [a.replace('pdschecksums', 'pdsinfoshelf') for a in sys.argv]
+        new_list = [a for a in new_list if a not in ('--infoshelf', '-i')]
+        status = os.system(' '.join(new_list))
```

Ruff (0.14.5)

Starting a process with a shell, possible injection detected

(S605)